frontend/package.json

```
@@ -11,6 +11,7 @@
   },
   "private": true,
   "dependencies": {
+    "@axios":"1.1.1",
```

```suggestion
    "@axios":"1.1.2",
```
this is a single comment with code suggestion